app/views/users/index.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Here are the users</h1>
\ No newline at end of file
+<h1> users posts</h1>
+<section>
+  <% @users.each do |user| %>
+    <div class="d-flex justify-content-between">
+      <div>
+        <img src="<%= user.photo %>" alt='User Avatar' style="width:20%" />
+      </div>
+      <div class='flex-grow-1 d-flex justify-content-between align-items-center'>
+        <a href="/users/<%= user.id %>">
+          <h1><%= user.name %></h1>
+        </a>
+        <p>Number of posts: <%= user.posts_counter || 0 %></p>
+      </div>
+    </div>
+  <%end%>
```

Comment:
- [ ] [REQUIRED] Great job so far @AlaaAlsalem , however, let's ensure that the user index page looks like the wireframe provided.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="346" alt="Screenshot 2023-06-24 at 19 57 37" src="https://github.com/AlaaAlsalem/blogApp/assets/58692590/7fa938b6-9733-4071-ad59-5d9f53a868a6"> | <img width="512" alt="Screenshot 2023-06-24 at 19 59 58" src="https://github.com/AlaaAlsalem/blogApp/assets/58692590/8e425c2a-3b65-47e0-89eb-52669bf9f6df">